src/spox/_future.py

```diff
@@ -221,5 +233,6 @@ def operator_overloading(
     "set_value_prop_backend",
     "value_prop_backend",
     # Operator overloading on Var
-    "operator_overloading",
+    "operator_overloading",  # noqa: F822
+    "__getattr__",
```

Comment:
```suggestion
```
`__all__` is only used for IDEs and when generating the docs. It has not impact on whether something can actually be imported. We don't want to keep deprecated things around in the docs, so it is fine to remove it here. We should also remove it from the actual docs [here](https://spox.readthedocs.io/en/latest/manual/unstable.html#operator-overloading) in a follow-up PR.

Reply:
> We should also remove it from the actual docs [here](https://spox.readthedocs.io/en/latest/manual/unstable.html#operator-overloading) in a follow-up PR.

Shouldn't this live in the PR removing the actual function?